modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -116,7 +116,7 @@ exports.oauthCallback = function (strategy) {
           return res.redirect('/authentication/signin');
         }
 
-        return res.redirect(redirectURL || sessionRedirectURL || '/');
```

Comment:
@fauria I already merged your commit with @OneOfTheWorld change.
If it's still not solving the problem, do you want to issue a PR that does?